docs/quick_start.md

```diff
@@ -0,0 +1,85 @@
+# Quick Start
+
+!!! Note
+    Currently, using the OSTIS Metasystem natively on Windows isn't supported.
+
+1. Clone repository of the OSTIS Metasystem:
+   
+    ```sh
+    git clone https://github.com/NikitaZotov/ostis-metasystem
+    cd ostis-metasystem
+    git checkout feat/update_build_system
+    git submodule update --init --recursive
+    ```
+
+2. Install C++ problem solver:
+
+    ```sh
+    ./scripts/install_cxx_problem_solver.sh
+    ```
+
+3. Build knowledge base:
+
+    ```sh
+    ./install/sc-machine/bin/sc-builder -i repo.path -o kb.bin --clear
+    ```
+
+4. Run C++ problem solver:
+
+    ```sh
+    ./install/sc-machine/bin/sc-machine -s kb.bin -c ostis-metasystem.ini \
+        -e "install/sc-machine/lib/extensions;install/sc-component-manager/lib/extensions;install/scp-machine/lib/extensions;install/problem-solver/lib/extensions"
+    ```
+
+5. Download component specifications from repositories. Execute the following command in a running terminal:
+
+    ```sh
+    components init
+    ```
+
+6. Install sc-web component. Execute the following command in the running terminal:
+
+    ```sh
+    components install sc_web
+    ```
+
+7. Build sc-web. Open new terminal and run:
+
+    ```sh
+    cd platform-dependent-components/interface/install/sc-web
+    npm run build
+    ```
+
+8. Run sc-web:
+   
+    ```sh
+    source .venv/bin/activate && python3 server/app.py --allowed_origins=http://localhost:3000
```

Comment:
this happens in logs and react-sc-web isn't working, it loads styles and containers, but no scn/buttons/labels
![image](https://github.com/user-attachments/assets/81040a50-9aae-4ff4-95ba-ff819a88b1f3)

Reply:
Fixed

Reply:
You'll see some interface incorrectness. For example, scn in sc-web will incorrectly display decomposition of myself, react-sc-web will not display sections. These problems are due to inconsistencies in the representation of relations and their processing. They will be fixed in the next pull requests. The display of scg-editor in react-sc-web will also be fixed in the next pull requests. I suggest to keep this pool request as it is, so as not to slow down the work of others

Reply:
All this didn't work before my pull request. We have a demo branch where this works through crutches